app/src/main/java/com/zulip/android/models/Message.java

```diff
@@ -104,7 +105,7 @@
     private String subject;
     @SerializedName("timestamp")
     @DatabaseField(columnName = TIMESTAMP_FIELD)
-    private Date timestamp;
+    private long timestamp;
```

Comment:
Changing the column fields isn't the best option, will it still work even for the old message table?